modules/popup/react/README.md

```diff
@@ -40,7 +40,11 @@ import {Popup} from '@workday/canvas-kit-react-popup';
 
 ## Static Properties
 
-> None
+#### `padding: PopupPadding`
```

Comment:
Since we're doing breaking changes anyway, can we change this from `padding` to `Padding`?